### DISCUSSION TITLE
[Feature] PXDisplayName, Regular Expression statements

### DISCUSSION THREAD

**Discussion by Amidweiz** [ikemen-engine/Ikemen-GO#844] [Feature Request] [answered]
P _X_ DisplayName would act identical to P _X_ Name except rather than reading the characters name, it would read their display name instead because sometimes characters are differentiated using display name rather than just name

Another suggestion is to be able to use regular expression statements within the P _X_ Name and P _X_ DisplayName rather than just matching for an exact name. It would cut down on needing multiple lines to read for one character

`Trigger1 = P2Name = "(a|A)kuma"`

**Comment by K4thos** [comment, marked as answer]
https://github.com/ikemen-engine/Ikemen-GO/wiki/Triggers-(new)#displayname-nightly-build-only
